libobs/CMakeLists.txt

```diff
@@ -19,12 +19,9 @@ add_subdirectory("${CMAKE_SOURCE_DIR}/deps/uthash" "${CMAKE_BINARY_DIR}/deps/uth
 
 target_sources(
   libobs
-  PRIVATE obs.c
-          obs.h
-          obs.hpp
-          obs-audio.c
-          obs-audio-controls.c
+  PRIVATE obs-audio-controls.c
```

Comment:
What's happening here? The sortable switch seems to have been applied here as well.

Reply:
Yesn't. The switch has been applied, but it also messes up directories and conditionals by mixing them together with normal files, which doesn't make much sense.

For example:
```
a.c
z.c
b.c
myfolder/whatever.c
$<$<BOOL:${SOME_VALUE}>:conditional.c>
```
would become
```
$<$<BOOL:${SOME_VALUE}>:conditional.c>
a.c
b.c
myfolder/whatever.c
z.c
```
which isn't desirable.

So as mentioned in the description, I added the flag, then undid the directory changes manually (and also had to of course remove the flag again). I don't think for libobs it's as bad not to have it as the lists aren't that volatile as in the UI, where files get added rather frequently in comparison.